### ISSUE TITLE
DE Begriffe Storystile

### ISSUE THREAD

**Issue by jansthh** [livingmydesign/hdstories#115]
- bei den Story Stilen verwendet er oft irritierende EN Begriffe 
- regelmäßig kommt es bei Alice vor 
- zB. die relevantesten: Cheshire Cat statt Grinsekatze oder Mad Hatter statt Hutmacher

**Comment by livingmydesign** [comment]
Ja. GPT hat eben die Daten aus einer englischen Quelle und nimmt an, dass die Namen der Charaktere Sondernamen sind. 

what to do..  man könnte da höchstens Dictionaries anlegen und zusätzlich einspeisen in den Prompt
